website/static/js/profile.js

```diff
@@ -483,17 +483,34 @@ var SocialViewModel = function(urls, modes) {
     TrackedMixin.call(self);
 
     self.addons = ko.observableArray();
+    
+    self.profileWebsites = ko.observableArray();
+
+    self.profileWebsitesLength = ko.computed(function() {
+        return self.profileWebsites().length;
+    });
+
+    self.hasProfileWebsites = ko.pureComputed(function() {
+        //Check to see if any valid profileWebsites exist
+        var profileWebsites = ko.toJS(self.profileWebsites());
+        for (var i=0; i<profileWebsites.length; i++) {
+            if (profileWebsites[i]) {
+                return true;
+            }
+        }
+        return false;
+    });
+
+    self.canRemove = ko.computed(function () {
+        if (self.profileWebsites()) {
```

Comment:
This can be condensed to `return self.profileWebsites() && self.profileWebsites().length > 1`